src/NUnitEngine/nunit.engine.tests.netstandard/nunit.engine.tests.netstandard.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>False</DelaySign>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <Version>$(PackageVersion)</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\CommonAssemblyInfo.cs" Link="Properties\CommonAssemblyInfo.cs" />
+    <Compile Include="..\EngineVersion.cs" Link="Properties\EngineVersion.cs" />
+    <Compile Include="..\nunit.engine.tests\Api\TestFilterTests.cs" Link="API\TestFilterTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Drivers\NotRunnableFrameworkDriverTests.cs" Link="Drivers\NotRunnableFrameworkDriverTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Internal\DirectoryFinderTests.cs" Link="Internal\DirectoryFinderTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Internal\PathUtilTests.cs" Link="Internal\PathUtilTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Internal\SettingsGroupTests.cs" Link="Internal\SettingsGroupTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Properties\AssemblyInfo.cs" Link="Properties\AssemblyInfo.cs" />
+    <Compile Include="..\nunit.engine.tests\Runners\MasterTestRunnerTests.cs" Link="Runners\MasterTestRunnerTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Services\ResultServiceTests.cs" Link="Services\ResultServiceTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Services\TestFilterBuilderTests.cs" Link="Services\TestFilterBuilderTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Services\TestFilteringTests.cs" Link="Services\TestFilteringTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Services\TestSelectionParserTests.cs" Link="Services\TestSelectionParserTests.cs" />
+    <Compile Include="..\nunit.engine.tests\Services\TokenizerTests.cs" Link="Services\TokenizerTests.cs" />
```

Comment:
Same [Seems like a glob could take care of almost all these items linked from the `..\nunit.engine` folder.]